imap_processing/tests/test_cli.py

```diff
@@ -80,7 +99,16 @@ def test_validate_args(instrument, data_level, raises_value_error):
 
 @mock.patch("imap_processing.cli.codice_l1a.process_codice_l1a")
 def test_codice(mock_codice_l1a, mock_instrument_dependencies):
-    """Test coverage for cli.CoDICE class"""
+    """
+    Test coverage for cli.CoDICE class.
+
+    Parameters
+    ----------
+    mock_codice_l1a : iterable TODO check
+        Mock codice l1a data TODO Check.
+    mock_instrument_dependencies : iterable TODO check
+        Mock instrument dependencies TODO check.
```

Comment:
```suggestion
    mock_codice_l1a : iterable
        Mock codice l1a data
    mock_instrument_dependencies : iterable
        Mock instrument dependencies
```

Can remove the TODOs for these because what you have looks good to me! (same for the other instruments)